toolkit/tools/grapher/grapher.go

```diff
@@ -26,6 +27,12 @@ var (
 	strictGoals      = app.Flag("strict-goals", "Don't allow missing goal packages").Bool()
 	strictUnresolved = app.Flag("strict-unresolved", "Don't allow missing unresolved packages").Bool()
 	timestampFile    = app.Flag("timestamp-file", "File that stores timestamps for this program.").String()
+	enableCpuProf    = app.Flag("enable-cpu-prof", "Enable CPU pprof data collection.").Bool()
```

Comment:
Another brainwave... why have all these flags listed here, we are going to turn around and immediately call `profile.StartProfiling()` with all of them in main and they are always the same for every tool. I wonder if we can just hide this all inside something like `exe.InputFlag(app, string)`

```go
// in exe.go
func ProfilingFlag(k *kingpin.Application)  *Profiler
	enableCpuProf = k.Flag("enable-cpu-prof", "Enable CPU pprof data collection.").Bool()
	enableMemProf    = k.Flag("enable-mem-prof", "Enable Memory pprof data collection.").Bool()
	enableTrace      = k.Flag("enable-trace", "Enable trace data collection.").Bool()
	cpuProfFile      = k.Flag("cpu-prof-file", "File that stores CPU pprof data.").String()
	memProfFile      = k.Flag("mem-prof-file", "File that stores Memory pprof data.").String()
	traceFile        = k.Flag("trace-file", "File that stores trace data.").String()

	p, err := profile.StartProfiling(*cpuProfFile, *memProfFile, *traceFile, *enableCpuProf, *enableMemProf, *enableTrace)
	if err != nil {
		// Unlcear what to do here, panic? Maybe kingping has a nice way to complain to the user?
	}
	return p
```

Reply:
Maybe we should treat this like we do the timestaping code, never break the program due to a timing failure. Just complain to the user and keep going with it turned off.

Reply:
This is a cleaner approach. Thanks